package-lock.json

```diff
@@ -0,0 +1,10041 @@
+{
```

Comment:
Correct me if I am wrong but shouldn't we have a yarn.lock instead of a package-lock.json? Or is this some yarn 4 thing? If so can we remove one of them?